src/main/thrift/parquet.thrift

```diff
@@ -881,15 +926,47 @@ struct ColumnChunk {
   /** Crypto metadata of encrypted columns **/
   8: optional ColumnCryptoMetaData crypto_metadata
```

Comment:
this field should be moved (or copied for backward compatibility) to a per-column struct that applies to all row groups / chunks. If we have such a struct.

Reply:
I removed the global struct.  There is now a new field on RowGroup that stores the ColumnChunks differently.